workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx

```diff
@@ -87,27 +87,27 @@ const WorkflowDefinitionDetailsCard = ({
   return (
     <InfoCard title="Details" className={classes.details}>
       <Grid container spacing={3} alignContent="flex-start">
-        <Grid container item md={4} spacing={3} alignContent="flex-start">
+        <Grid container item md={12} spacing={3} alignContent="flex-start">
```

Comment:
There is no need anymore for two hierarchies of the grid, you can move the inner one